modules/HardwareDrivers/PowerMeters/CarloGavazzi_DCT1/meter/transport.hpp

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+
+#ifndef POWERMETER_TRANSPORT_HPP
+#define POWERMETER_TRANSPORT_HPP
+
+/**
+ * Baseclass for transport classes.
+ *
+ * Transports are:
+ * - direct connection via modbus
```

Comment:
The direct connection part of this is probably stripped?

(This abstraction could have provided a nice EVerest library, in `lib/`.)